aptos-move/framework/aptos-stdlib/sources/cryptography/multi_ed25519.move

```diff
@@ -191,47 +222,62 @@ module aptos_std::multi_ed25519 {
         message: vector<u8>
     ): bool;
 
+    #[test_only]
+    native fun generate_keys_internal(total: u8, threshold: u8): (vector<u8>,vector<u8>);
+
+    #[test_only]
+    native fun sign_internal(sk: vector<u8>, message: vector<u8>): vector<u8>;
+
     //
-    // Test cases
+    // Tests
     //
 
+    #[test_only]
+    struct TestMessage has copy, drop {
+        foo: vector<u8>,
+        bar: u64,
+    }
+
+    #[test_only]
+    fun pollute_first_signature(sig: &mut Signature) {
```

Comment:
Rename to `maul_first_signature` and make it `public`, since we might wanna call it in other modules when testing.